web.js

```diff
@@ -91,8 +91,8 @@ app.route('/:user/:repo/:branch/*')
         middleware.accessControl
     )
     .get(
-        middleware.fileRedirect('https://raw.githubusercontent.com'),
-        middleware.proxyPath('https://raw.githubusercontent.com')
+        middleware.fileRedirect(config.gitDomain || 'https://gist.githubusercontent.com'),
```

Comment:
This will result in GitHub repo files being requested from `gist.githubusercontent.com`, which won't work.